CONTRIBUTING.md

```diff
@@ -43,3 +43,10 @@ We are always looking for the quality contributions and will be happy to accept
   * your directive has accompanying tests and all the tests are passing; don't hesitate to contact us (angular-ui@googlegroups.com) if you need any help with unit testing
   * your PR doesn't break the build; check the Travis-CI build status after opening a PR and push corrective commits if anything goes wrong
   * your commits conform to the conventions established [here](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md)
+* If you have an idea for, or the code you want to contribute is, a new feature, or changes the way an existing feature works (possibly a breaking change), please open an issue to start the discussion so that we may collaborate on an acceptable solution if one exists.
+  * unless it's well thought out or correcting a small error (e.g. a typo in the docs) or we would prefer you not submit a PR with the new feature or change without first discussing with us - we value your time and this is to ensure you aren't wasting it.
```

Comment:
I would avoid contractions such as `it's` and `aren't` here, but perhaps I'm being a little pedantic - avoiding contractions is viewed as more professional.